__tests__/kata1.fizzBuzz.test.js

```diff
@@ -2,18 +2,21 @@ const { fizzBuzz } = require('../src');
 
 describe('fizzBuzz', () => {
   it('returns Fizz when passed a multiple of 3', () => {
-
+        expect(fizzBuzz(3)).toBe('Fizz')
```

Comment:
6 space indentation 😱😱😱😄. Ensure consistent indentation across files, and only ever indent either 2 spaces or 4 spaces. I suggest 2 spaces as this is more widely adopted.